content/en/docs/concepts/logic.md

```diff
@@ -642,6 +671,11 @@ that do not depend on each other.
 
 The values lattice brings CUE another advantage: the ability to reason about
 values, schema, and constraints.
+<!-- (jba) I think "schema" is confusing as the plural. Choose either "schemas"
+or "schemata". I think the former is, sadly, preferable to modern ears.
+(This applies to several instances of "schema" above too.) 
+-->
```

Comment:
What's next, a bottoms-up model? I prefer schemata, but you are probably right about shemas being better.